src/site/lab11.rst

```diff
@@ -0,0 +1,110 @@
+*******
+Lab #11
+*******
+
+* Refer to the topic pages for this lab
+    * You will find many of the answers to your questions there
+* Feel free to use your laptop if you have it
+* I strongly encourage you to work with others in the lab
+* Ensure I have recorded your completion. Failure to do so will result in a grade of 0
+* When you get stuck, do me a favour and ask those sitting around you for help
+    * I want people to get used to working together in the labs
+    * Peer teaching and peer learning is super effective
+
+
+Starting Point
+==============
+
+This lab focuses on the sorts discussed in :doc:`topic 26 </topic26>`.
+
+.. code-block:: java
+
+    public static void main(String[] args){
+
+        IndexedBag<Integer> toSort = createUnsortedIntBag(10000, 10);
+        System.out.println(toSort);
+
+        double startTime = System.nanoTime();
+        //quicksort(toSort);
+        mergesort(toSort);
+        double endTime = System.nanoTime();
+
+        System.out.println(toSort);
+        System.out.println("runtime: " + (endTime - startTime)/1000000 + " ms");
```

Comment:
Instead of commenting a sort out, maybe just always run and time both sorts? A little less effort to get it up and running.